colorizer_data/converter.py

```diff
@@ -401,7 +402,7 @@ def convert_colorizer_data(
     frames_3d: Optional[Frames3dMetadata] = None,
     centroid_x_column: str = "Centroid X",
     centroid_y_column: str = "Centroid Y",
-    centroid_z_column: Optional[str] = None,
+    centroid_z_column: Optional[str] = "Centroid Z",
```

Comment:
Random thought: I would consider just always having a z coordinate and let it be 0 in the 2D case...

Reply:
We also need to keep backwards compatibility with all the old 2D datasets that don't have a Z-coordinate either...